### PR TITLE
Update some packages

### PR DESCRIPTION
**Pull request by Guillaume-meta**
See package.json

### CODE REVIEW

**Comment by Guillaume-meta** [comment]
Someone could fix php error please ? Don't understand what it want from me :)

File: template/web/wp-content/themes/<%= slug %>/app/Managers/TwigManager.php [hunk 1]
```diff
@@ -16,12 +17,38 @@ class TwigManager implements ManagerInterface {
 	 */
 	public function run() {
 		add_filter( 'timber/twig', array( $this, 'add_twig_toolkit' ) );
```
Comment (titouanmathis) [inline template/web/wp-content/themes/<%= slug %>/app/Managers/TwigManager.php]: **nitpick:** Remove the `add_twig_toolkit` hook

The `studiometa/ui` Twig extension extends the `studiometa/twig-toolkit` one, so we do not need to register it here anymore.

See https://github.com/studiometa/ui/blob/develop/packages/twig-extension/Extension.php#L19

**Review by titouanmathis — CHANGES_REQUESTED** [review]

**Comment by perruche** [comment]
CI is fix. WIth the upgrade to studiometa/webpack-config, the tests need to be run on node 16.

File: template/package.json [hunk 2]
```diff
@@ -21,23 +21,23 @@
...
     "prettier-plugin-twig-melody": "^0.4.6",
     "stylelint": "^13.13.1",
     "stylelint-formatter-gitlab": "^1.0.1",
-    "tailwindcss": "^2.1.19"
+    "tailwindcss": "^3.0.23"
   },
   "dependencies": {
-    "@studiometa/js-toolkit": "^2.0.0-beta.0",
-    "@studiometa/ui": "^0.1.0"
+    "@studiometa/js-toolkit": "^2.0.1",
```
Comment (perruche) [inline template/package.json]: Can we update to the latest version: 2.2.2 ? This should not have any impact right @titouanmathis ?

**Review by titouanmathis — APPROVED** [review]

**Pull request merged**